apps/condo/domains/notification/constants/constants.js

```diff
@@ -651,9 +651,15 @@ const MESSAGE_TYPES = Object.keys(MESSAGE_META)
 
 const MESSAGE_DELIVERY_STRATEGY_AT_LEAST_ONE_TRANSPORT = 'atLeastOneTransport'
 const MESSAGE_DELIVERY_STRATEGY_ALL_TRANSPORTS = 'allTransports'
+
 const MESSAGE_DELIVERY_DEFAULT_PRIORITY = 'default'
 const MESSAGE_DELIVERY_SLOW_PRIORITY = 'slow'
 const MESSAGE_DELIVERY_FAST_PRIORITY = 'fast'
+const MESSAGE_DELIVERY_PRIORITY_TO_TASK_QUEUE_MAP = {
+    [MESSAGE_DELIVERY_FAST_PRIORITY]: 'high',
+    [MESSAGE_DELIVERY_DEFAULT_PRIORITY]: 'medium',
+    [MESSAGE_DELIVERY_SLOW_PRIORITY]: 'low',
```

Comment:
Why not just priority as an integer value (a higher value is more prioritized)?
What if we will need something between medium and high?
How these 3 queues will work?

Reply:
That's our architectural decision. Before we used to specify numeric priority. Which is (probably) powered by [priority queue](https://en.wikipedia.org/wiki/Priority_queue) data structure. This solution usually requires `O(logN)` to insert element and not even supported well outside of Redis provider.

The other popular way is to use N named queues. So you can create common worker to get tasks from all of them. Or worker for a specific queue (for example, dedicated worker for `high`-queue tasks, which will process data even if all other workers are full)